grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -368,7 +372,12 @@ private void startLbRpc() {
     checkState(lbStream == null, "previous lbStream has not been cleared yet");
     LoadBalancerGrpc.LoadBalancerStub stub = LoadBalancerGrpc.newStub(lbCommChannel);
     lbStream = new LbStream(stub);
-    lbStream.start();
+    Context baseContext = context.attach();
```

Comment:
This is not a baseContext. Rename to `prevContext` or similar? Ditto for the test.

Reply:
Fixed.